listings/applications/l1_l2_token_bridge/solidity/lib/forge-std/.gitattributes

```diff
@@ -0,0 +1 @@
+src/Vm.sol linguist-generated
```

Comment:
A couple of notes/questions:
- I added tests for the Solidity contracts to ensure correctness, and opted for Foundry for consistency (as we already use it for Starknet contract tests). If you think this adds to many other files (see next point), and don't think they're that necessary for this example, we can delete.
- `forge-std` library is included in the source, as it makes it any extra setup unnecessary to run tests for the Solidity `TokenBridge` in the future. Let me know if you want this removed, and we can add a README note with instructions on how to set this up.
- I didn't update _./scripts/cairo_programs_verifier.sh_ to run Solidity tests, but this can be done.